recipes/targetscan/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "targetscan" %}
+{% set version = "7.0" %}
+{% set sha256 = "6e23c72a920f2a58ecf783fe33a6929bc16bbeb5359b020505c0fc95019bd749" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://www.targetscan.org/vert_80/vert_80_data_download/targetscan_70.zip
```

Comment:
The URI should refer to the version - you can change it to remove the full stop- see [example](https://github.com/bioconda/bioconda-recipes/blob/57e2c903407676cca776299a9e0bf82036c0631a/recipes/r-shinyngs/meta.yaml#L6).